include/oneapi/dpl/pstl/parallel_impl.h

```diff
@@ -66,6 +66,40 @@ __parallel_find(_ExecutionPolicy&& __exec, _Index __first, _Index __last, _Brick
     return __extremum != __initial_dist ? __first + __extremum : __last;
 }
 
+template <class _ExecutionPolicy, class _Index, class _Brick, class _IsFirst>
+_Index
+__parallel_find(__tbb_backend_tag __tag, _ExecutionPolicy&& __exec, _Index __first, _Index __last, _Brick __f, _IsFirst)
```

Comment:
It seems that here should be the "common" backend tag, not specific. Because, this file contains the patterns shared between the available parallel backends.

Reply:
Fixed